src/main/java/com/synopsys/integration/detect/lifecycle/run/RunResult.java

```diff
@@ -49,18 +48,9 @@ public void addDetectCodeLocations(final List<DetectCodeLocation> codeLocations)
 
     public void addDetectableToolResult(final DetectableToolResult detectableToolResult) {
         detectableToolResult.getDetectToolProjectInfo().ifPresent(detectToolProjectInfo1 -> addToolNameVersion(detectToolProjectInfo1.getDetectTool(), detectToolProjectInfo1.getSuggestedNameVersion()));
-        detectableToolResult.getDockerTar().ifPresent(this::addDockerFile);
         detectCodeLocations.addAll(detectableToolResult.getDetectCodeLocations());
     }
 
-    public void addDockerFile(final File dockerFile) {
-        dockerTar = dockerFile;
-    }
-
-    public Optional<File> getDockerTar() {
-        return Optional.ofNullable(dockerTar);
-    }
-
```

Comment:
Regarding naming of these methods and variables: I think the relevant bits about this file are (a) it contains the image the user gave us, and (b) it contains a docker image (saved as a tar). So maybe givenImage, givenImageTar, or givenImageFile or similar? Also, I would think we'd want the add and get methods for it to be named consistently so it's clear they are adding/getting the same thing.

Reply:
@stevebillings I like providedImageTar as an improvement.  Are you talking about consistency with RunResult, because I removed the dockerTar field form that class altogether, as it is no longer needed.

Reply:
oh jeez, yeah. NVM!